index.html

```diff
@@ -320,12 +327,12 @@ <h4>Multikey</h4>
 
           <p>
 The `publicKeyMultibase` property represents a Multibase-encoded Multikey
-expression of a Dilithium public key.
-<span class="issue">Specify acceptable Dilithium key sizes</span>
+expression of a ML-DSA public key.
+<span class="issue">Specify acceptable ML-DSA key sizes</span>
           </p>
 
           <p>
-The Multikey encoding of a Dilithium public key MUST start with the two-byte
+The Multikey encoding of a ML-DSA public key MUST start with the two-byte
 prefix `0x8724` (the varint expression of `0x1207`) followed by the 1312-byte
```

Comment:
Reminder that we /really/ need to register these values here: https://github.com/multiformats/multicodec/blob/master/table.csv#L168

Reply:
I will raise an issue